components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -612,6 +612,16 @@ abstract class AbstractFetchDownloadService : Service() {
     private fun sendDownloadStopped(downloadState: DownloadJobState) {
         downloadState.notifiedStopped = true
 
+        /**
+         * In cases when we don't get the file size provided initially, we have to
+         * use downloadState.currentBytesCopied, as a fallback.
+         */
+        val fileSizeNotFound = downloadState.state.contentLength == null || downloadState.state.contentLength == 0L
+        if (downloadState.status == COMPLETED && fileSizeNotFound) {
```

Comment:
@csadilek when the download is completed and we don't have a file size we should try to guess the file size with the information that we have to avoid this [issue](https://github.com/mozilla-mobile/fenix/issues/11107#issuecomment-646077473) (Fenix it's using it in their UI)

Reply:
OK, yeah, seems like a reasonable workaround to use bytes copied as total size if the download is completed and we don't have a content length...